README.md

```diff
@@ -1,45 +1,98 @@
+|               |               IPLD Data Model               |   ipld-zcash   |          |
+|               |                                             |                |          |
++-------------------------------------------------------------+----------------+----------+
+                |                                                                         |
+                |                CID                      Path                            |
+                |                                                                         |
+                +-------------------------------------------------------------------------+
+```
 
-- ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) - this spec is a work-in-progress, it is likely not even complete.
-- ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) - this spec is a rough draft and will likely change substantially.
-- ![](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square) - this spec is believed to be close to final. Minor changes only.
-- ![](https://img.shields.io/badge/status-stable-brightgreen.svg?style=flat-square) - this spec is likely to improve, but not change fundamentally.
-- ![](https://img.shields.io/badge/status-permanent-blue.svg?style=flat-square) - this spec will not change.
+The goal of this stack is to enable decentralized data-structures
+which in turn will enable more decentralized applications.
 
-Nothing in this spec repository is `permanent` yet. As in many IPLD repositories, most of the work is happening in [the issues](https://github.com/ipld/specs/issues/) or in [active pull requests](https://github.com/ipld/specs/pulls/). Go take a look!
+Many of the specifications in this stack are inter-dependent.
 
-## Documents
+```
+       IPLD Dependency Graph
 
-- [**Roadmap**](/ROADMAP.md)
-- **Specifications:**
-  - ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) [`IPLD`](/IPLD.md) - spec about the data model, pointers and link formats
-  - ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) `IPLD Selectors` - spec about simple language to select multiple unknown nodes in a graph
-  - ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) `IPLD Transformations` - spec about the language to trasform an IPLD graph into another
-  - ![](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square) [`CID (Content IDentifier)`](https://github.com/ipld/cid)
-  - ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) [`IPLD Formats`](https://github.com/ipld/interface-ipld-format) - interface definition for adding support to different formats
-  - ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) [`CAR`](/CAR.md) - Content Addressable Archives
++---+                          +-----+        +---------+
+|CID+-----------+-------------->Block+-------->Raw Block|
```

Comment:
I think you explained it to me two times already. I still don't get the direction of the arrow. But I don't want to block things because of such a small thing. If everyone else gets the idea, that's good enough for me :)

Reply:
It's like a reverse dependency graph. One thing is required to build the next thing. The only purpose of the "Raw Block" illustration is to note that raw blocks don't have a codec but are still part of replication.

Reply:
Oh, I think I got it now. You mean a "Raw Block" as a "block with the raw codec". It doesn't contain links, hence it doesn't belong to the other codecs. So it could also be drawn like this:

```
+---+                                        +-----+
|CID+-----------+--------------------+-------+Block|
+---+           |                    |       +-----+
        +-------v------------+  +----v----+
        |Links (Conceptually)|  |Raw Block|
+----+  +-------+------------+  +--+------+ 
|Path|          |                  |          +-----------+
+----+          |                  +---------->Replication|
  |    Codecs   |                  |          +-----------+
+-v-------------v------------------+--+
|                                     |
```